### ISSUE TITLE
Feature Request: Ability to import Payload Configs

### ISSUE THREAD

**Issue by checkymander** [its-a-feature/Mythic#117]
It would be cool if we could have the ability to import payload configs in the UI, so if we needed to switch servers for any reason any payloads that are active on target would still be able to call back.

**Comment by its-a-feature** [comment]
You can currently export payload configs and import payload configs as a means to generate new payloads more easily, but it currently doesn't allow you to essentially move data between servers. 

In your use case, you have another Mythic server that you want to also be able to catch the same Mythic agents?

**Comment by checkymander** [comment]
Yep basically.

**Comment by its-a-feature** [comment]
The tough part there is that Mythic tracks more than just the payload UUID when it comes to catching a callback. Mythic needs to know:

- Backing PayloadType, PayloadType's build parameters, and the selected values in this payload for each one of those
- Commands selected to be included in the payload
- C2 profiles selected and the value selected for each one of their parameters 

Trying to map and correlate all of that information into another database isn't easy. There are even more things you have to track and carry over if you wanted to be able to continue using existing callbacks when you swap the server. 

Is the scenario here one where you don't have time to save the current database, swap out the server, and put the database back?

**Comment by checkymander** [comment]
Specifically 2.1 -> 2.2 where we wouldn't be able to save the DB

**Comment by checkymander** [comment]
So it looks like the feature I was looking for actually seems to already exist lol. I just wanted to import the config of an existing created payload (UUID, Encryption Key, etc etc), and it seems to already be possible.

**Comment by checkymander** [comment]
I'm actually going to re-open this, as it seems you can't actually import encryption keys, and all that fun stuff.

**Comment by checkymander** [comment]
tl;dr our db became corrupted, and we wanted to try and recover some of the callbacks we already had when we did the database reset, that's where this pull request came from.

**Comment by its-a-feature** [comment]
corrupted how? when you do the database reset you are deleting all the data in the database
